systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceManager.java

```diff
@@ -265,6 +266,33 @@ public static HelmClient helmClient() {
         return resourceReady[0];
     }
 
+    /**
+     * Auxiliary method for copying Controller {@link Constants.TEST_SUITE_NAME_LABEL} and {@link Constants.TEST_CASE_NAME_LABEL} labels
```

Comment:
What makes these labels _controller_ labels? What is the controller?

Reply:
I think I could eliminate `Controller` and just enumerate `TEST_SUITE_NAME_LABEL` and `TEST_CASE_NAME_LABEL` labels. The original idea was that Controller is (Job, Deployment etc.) so that's why controller labels.